SR2024/2024-03-13-competition-rsvp-chase.md

```diff
@@ -0,0 +1,21 @@
+---
+to: SR2024 teams who still haven't RSVP'd
+subject: SR2024 RSVP & check-in
+---
+
+Hi $TODO_NAME,
+
+{# either #}
+It was great to see your team at the Virtual Competition this weekend.
+{# or #}
+We missed you at the Virtual Competition this weekend.
+{# / #}
+
+I wanted to check on how you're doing and whether you wanted any extra help -- via Discord, in person or otherwise. There's still plenty of time to work on your robot before the competition!
+
+We'd also like to finalise our planning for the competition, including things like assigning team pits at the venue -- for which we need to know a little about the competitors you'll be bringing.
+
+**Please could you [fill out your team details](https://forms.gle/8YhfdRbfDWP3KKNy8) to let us know.**
+
+Thanks,
+Peter
```

Comment:
Question: Is the intention for this to come from you personally, vs the normal "Competition Team" sign off?

Reply:
From me -- I'm planning to just send these as separate emails, nominally from me though still from teams@.